### DISCUSSION TITLE
Removing global metadata field

### DISCUSSION THREAD

**Discussion by sarah-beasley** [open-sdg/open-sdg#1994] [Q&A] [answered]
I am having some trouble trying to remove the Global indicator description field from the Global metadata tab. 

We are not using the hide_empty_metadata functionality, but are using the empty_metadata_placeholder functionality instead. But for every single indicator this Global indicator description field is empty so always shows 'not available for this indicator'. Therefore we would like to remove this field from appearing completely in the tab, however everything I have tried so far has not worked. 

In the prose.yml file, I have tried replacing:
      - name: "indicator_definition"
        field:
            element: textarea
            label: "Global indicator description"
            scope: global
with:
      - name: "indicator_definition"
        field:
            element: hidden
            label: "Global indicator description"
            scope: global

And I have also tried deleting these lines completely too. Neither resulted in the description field disappearing from the global metadata tab.

Feature branch in [GitHub](https://github.com/ONSdigital/sdg-data/blob/feature-remove-global-description/_prose.yml)
Feature branch to [see change](http://uk-sdg-feature-branches.s3-website.eu-west-2.amazonaws.com/feature-remove-global-description/3-1-1/)

**Comment by brockfanning** [comment, marked as answer]
@sarah-beasley I would have expecting deleting the entire field from _prose.yml to have worked. In other words deleting all of these lines:

```
- name: "indicator_definition"
  field:
    element: textarea
    label: "Global indicator description"
    scope: global
```

Can you give that another try?

**Reply by sarah-beasley** [reply]
that's so bizarre, I've just tried again and it's worked now! Magic, thank you
